resources/lang/en/forms.php

```diff
@@ -26,5 +26,6 @@
         'drag_drop_browse' => '<span class="hidden md:inline">Drag & Drop your files here or</span> <a @click="select()" class="link">Browse Files</a>',
         'delete_image'     => 'Delete Image',
         'requirements'     => '<span>Min size :width x :height. Max filesize :filesize.</span> <span>Max :quantity images</span>',
+        'requirements_no_max'     => '<span>Min size :width x :height. Max filesize :filesize.</span>',
```

Comment:
I don't think we will ever need to support unlimited entries 🤔 at least not in the current products so you can remove that feature

Reply:
@ItsANameToo ok ill default to 8 again then